vipermonkey/core/expressions.py

```diff
@@ -369,22 +443,21 @@ def eval(self, context, params=None):
 # operator (if present) is right-associative. Any assignment operators are
 # also typically right-associative."
 
-expression <<= infixNotation(expr_item,
-                             [
-                                 # ("^", 2, opAssoc.RIGHT), # Exponentiation
-                                 # ("-", 1, opAssoc.LEFT), # Unary negation
-                                 ("*", 2, opAssoc.LEFT, Multiplication),
-                                 ("/", 2, opAssoc.LEFT, Division),
-                                 ("\\", 2, opAssoc.LEFT, FloorDivision),
-                                 (CaselessKeyword("mod"), 2, opAssoc.RIGHT, Mod),
-                                 ("+", 2, opAssoc.LEFT, Sum),
-                                 ("-", 2, opAssoc.LEFT, Subtraction),
-                                 ("&", 2, opAssoc.LEFT, Concatenation),
-                                 # (CaselessKeyword("xor"), 2, opAssoc.LEFT),
-                             ])
+expression <<= (operatorPrecedence(expr_item,
```

Comment:
according to pyparsing's doc, operatorPrecedence is deprecated, should be replaced by infixNotation.

Reply:
Fixed in commit c963b624223cefed44fbb08bf7e70bbca0a6bf54.